src/components/common/image-preview.js

```diff
@@ -67,8 +67,26 @@ const StyledImagePreview = styled.div.attrs({
   }
 `;
 
-const ImagePreview = ({exportImage = {}, width = 400, showDimension}) => {
-  const {error, imageDataUri, exporting, imageSize: {imageW, imageH} = {}} = exportImage;
+/** @typedef {{
```

Comment:
`ExportImage` is defined in `ui-state-updater.d.ts`

Reply:
Great. Will update to use that in #1190